src/main/java/com/example/test/sorters/Sorter.java

```diff
@@ -0,0 +1,33 @@
+package com.example.test.sorters;
+
+import com.example.test.R;
+
+public interface Sorter {
+
+    int[] sort(int[] array);
+
+    enum SortingMethod { // binded with strings.xml for changes
```

Comment:
Each item has own binding to string resource
```suggestion
    enum SortingMethod {
```